vmcloak/dependencies/win7sp.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2014-2016 Jurriaan Bremer.
+# This file is part of VMCloak - http://www.vmcloak.org/.
+# See the file 'docs/LICENSE.txt' for copying permission.
+
+import os.path
+import time
+
+from vmcloak.abstract import Dependency
+
+class Win7sp(Dependency):
+    name = "win7sp"
+    exes = [
+        {
+            "version": "1",
+            "target": "win7x86",
```

Comment:
Not tested, but there might be a typo... According to the file name (`windows6.1-KB976932-X64.exe`), it's related to `x64`. However, `target` is set to `win7x86`.